server/routes/testutils/appSetup.ts

```diff
@@ -36,15 +36,23 @@ const bookerReference = TestData.bookerReference().value
 
 export const flashProvider = jest.fn()
 
-function appSetup(services: Services, production: boolean, userSupplier: () => Express.User): Express {
+function appSetup(
+  services: Services,
+  production: boolean,
+  userSupplier: () => Express.User,
+  sessionData: SessionData,
+): Express {
   const app = express()
 
   app.set('view engine', 'njk')
 
   nunjucksSetup(app, testAppInfo)
-  app.use(cookieSession({ keys: [''] }))
```

Comment:
Removing `cookie-session` as it was interfering with simply passing through an object to mock `req.session` - and it didn't appear to be doing anything useful.

Reply:
Where is the session stored if this is removed?

Reply:
Basically just a `SessionData` object (defaulting to `{ }`) is now passed through so data can be seeded before requests and stuff can be 'saved' into it. 

I guess without a 'proper' module like `cookie-session` we're not getting any of the things like `session.destroy()` - but nothing like that gets called. (Auth code does, but that's not included in this test app setup; it just assumes a logged in user.)

In Staff UI we removed this too. I think in the Admin UI its still there - but we don't use `req.session` so doesn't matter.